<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widgetScreen</class>
 <widget class="QWidget" name="widgetScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1962</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widgetScreen_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1881</width>
     <height>741</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>0</y>
      <width>641</width>
      <height>741</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="MpColorPlot" name="lofarcvPlot" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>631</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="MpColorPlot" name="lofarcoPlot" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>631</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="MpColorPlot" name="demoncoPlot" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>570</y>
       <width>631</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="MpColorPlot" name="demoncvPlot" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>470</y>
       <width>631</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="echoTxt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>631</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="curvePlot" name="lofarcvPlotcurve" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>631</width>
       <height>91</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="curvePlot" name="demoncvPlotcurve" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>631</width>
       <height>91</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1241</width>
      <height>741</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="MpColorPlot" name="pasPlot" native="true">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>200</y>
       <width>1261</width>
       <height>541</height>
      </rect>
     </property>
    </widget>
    <widget class="MpColorPlot" name="cpPlot" native="true">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>0</y>
       <width>1261</width>
       <height>0</height>
      </rect>
     </property>
    </widget>
    <widget class="curvePlot" name="cpPlotcurve" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1241</width>
       <height>191</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MpColorPlot</class>
   <extends>QWidget</extends>
   <header location="global">mpcolorplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>curvePlot</class>
   <extends>QWidget</extends>
   <header location="global">curveplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
